### PR TITLE
bump: Update tool to version 2.2.0 CY-4974

### PR DESCRIPTION
**Pull request by lucassklp**

### CODE REVIEW

File: src/main/scala/codacy/codenarc/CodeNarcOutput.scala [hunk 1]
```diff
@@ -15,7 +16,7 @@ object CodeNarcOutput {
         case violation: Violation =>
           CodeNarcOutput(
             fileResults.getPath,
-            violation.getMessage,
+            Option(violation.getMessage).getOrElse("Not available"),
```
Comment (lolgab) [inline src/main/scala/codacy/codenarc/CodeNarcOutput.scala:19]: `"Not available"` ?
Doesn't make more sense an empty string ?
Also, do we know if Codacy allows to undefined `message`s for Patterns ?

**Review by lolgab — CHANGES_REQUESTED** [review]

File: src/main/scala/codacy/codenarc/CodeNarcOutput.scala (hunk 1 above)
Comment (machadoit) [reply src/main/scala/codacy/codenarc/CodeNarcOutput.scala:19]: I would really prefer a "N/A" or "not available" then an empty string. Actually I prefer just "N/A".

I am a bit concern with an empty string making something fail as well, and then we assume that this didn't fix anything...

File: src/main/scala/codacy/codenarc/CodeNarcOutput.scala [hunk 2]
```diff
@@ -2,6 +2,7 @@ package codacy.codenarc
 
 import org.codenarc.results.{DirectoryResults, FileResults, Results}
 import org.codenarc.rule.Violation
+import scala.Option
```
Comment (lolgab) [inline src/main/scala/codacy/codenarc/CodeNarcOutput.scala:5]: This is already [imported automatically](https://www.scala-lang.org/api/2.13.6/scala/Predef$.html) by `Predef.scala`

**Review by lolgab — CHANGES_REQUESTED** [review]

**Review by machadoit — APPROVED** [review]

File: src/main/scala/codacy/codenarc/CodeNarcOutput.scala [hunk 3]
```diff
@@ -15,7 +16,7 @@ object CodeNarcOutput {
         case violation: Violation =>
           CodeNarcOutput(
             fileResults.getPath,
-            violation.getMessage,
+            Option(violation.getMessage).getOrElse(violation.getRule.getName),
             violation.getRule.getName,
```
Comment (lolgab) [inline src/main/scala/codacy/codenarc/CodeNarcOutput.scala:19]: Can you please name a variable before like `val ruleName = violation.getRule.getName` and use that ?

**Review by lolgab — CHANGES_REQUESTED** [review]
Last nitpick

**Review by lolgab — APPROVED** [review]

**Pull request merged**